ui/src/components/filter/configurations/kvFilter.ts

```diff
@@ -0,0 +1,12 @@
+import {FilterConfiguration} from "../../../components/filter/utils/filterTypes";
+import {useI18n} from "vue-i18n";
+
+export const kvsFilter = (): FilterConfiguration => {
```

Comment:
The name of the function should match the file name.

```suggestion
export const kvFilter = (): FilterConfiguration => {
```